Libs/GObject/Structs/Type.cs

```diff
@@ -49,6 +49,25 @@ public Type(ulong value)
 
         #endregion
 
+        // Print out the name of the gtype (for debugging purposes)
+        public override string ToString()
+        {
+            return Marshal.PtrToStringAnsi(Global.Native.type_name(_value));
+        }
+
+        public override bool Equals(object obj)
+        {
+            if (obj is Type && ((Type)obj)._value == _value)
+                return true;
+
+            return false;
+        }
+
+        public override int GetHashCode()
```

Comment:
This is doing nothing

Reply:
This is needed for the implementation of `Equals`. Since we're not doing that anymore, it can be safely removed.